node/test/internalhelpertests.ts

```diff
@@ -401,4 +401,10 @@ describe('Internal Path Helper Tests', function () {
 
         done();
     });
+    
+    it('ReportMissingLocalization', (done: MochaDone) => {
```

Comment:
@cima this test does not really new changes - it passes successfully with and without new changes.

Reply:
You are right that the test finishes OK regardles off my change. The propper testing would require some serious mocking using rewire mock, which would require new dependencies. Maybe damaging `__dirname` variable within the unittest might do the trick. It is bit strettching beyond replacing two strings. Shall I fully simulate the situation when no file is loaded?

Reply:
@cima yes, I guess that would be more helpful for coverage purposes.

Reply:
I have included a test that simulates the missing file. When such test (5da2a2a98428d6300bde4e529237ad181c1bd2a1) is applied to master (ba1f94be19f958f1f718ac946aef3123c8eee4a8) it triggers the infinite recusion.

Reply:
@cima thanks a lot!